packages/tinacms/src/tina-cms.tsx

```diff
@@ -245,23 +240,7 @@ export const TinaCMSProvider2 = ({
         schema={{ ...schema, config: { ...schema.config, ...props } }}
       >
         <style>{styles}</style>
-        <ErrorBoundary>
-          <DocumentCreator documentCreatorCallback={documentCreatorCallback} />
-          <TinaDataProvider formifyCallback={formifyCallback}>
```

Comment:
It is this change here that is breaking the tests. The example is out of date and is still using the "old way". I can update the example